ddtrace/contrib/cherrypy/middleware.py

```diff
@@ -68,11 +67,11 @@ def _setup(self):
         cherrypy.request.hooks.attach("after_error_response", self._after_error_response, priority=5)
 
     def _on_start_resource(self):
-        if self.use_distributed_tracing:
-            context = HTTPPropagator.extract(cherrypy.request.headers)
-            # Only need to activate the new context if something was propagated
-            if context.trace_id:
-                self._tracer.context_provider.activate(context)
+        trace_utils.activate_distributed_headers(
+            self._tracer,
+            request_headers=cherrypy.request.headers,
+            override_distributed_tracing=self.use_distributed_tracing,
```

Comment:
I think we can get rid of the `use_distributed_tracing` property on this class since all it does is refer to the config object - which the helper does